src/vcs/git/object.rs

```diff
@@ -126,6 +126,16 @@ impl BranchName {
     }
 }
 
+/// Stats for a repository
+pub struct Stats {
+    /// Number of commits
+    pub commit_count: usize,
+    /// Number of branches (local & remote)
```

Comment:
I think we want to be using only local branches for this stat, otherwise it'll be confusing. You can do this with `    list_branches(Some(BranchType::Local))`

Reply:
Thanks, we weren't sure what would make sense so we defaulted to all :) But `Local` makes sense :+1: